magefile.go

```diff
@@ -251,6 +263,21 @@ func TestSmoke() error {
 	return shx.Command("go", "test", "-count=1", "-timeout=20m", "-tags", "smoke", v, "./tests/smoke/...").CollapseArgs().RunV()
 }
 
+// Run grpc service tests
+func TestGRPCService() {
+	var run string
+	runTest := os.Getenv("PORTER_RUN_TEST")
```

Comment:
It looks like we are copying a similar idea as the integration tests - is this so we don't accidentally run this or so we only run this with integration tests?

Reply:
It allows setting an env var to run a specfic test by name.  Just carrying forward the pattern from integration tests.